test/e2e_node/eviction_test.go

```diff
@@ -357,9 +357,6 @@ var _ = SIGDescribe("PriorityMemoryEvictionOrdering [Slow] [Serial] [Disruptive]
 			memoryConsumed := resource.MustParse("600Mi")
```

Comment:
the changes to this file don't seem related to the readonly port change, should this file be reverted?

Reply:
reverted.